src/actions/base.ts

```diff
@@ -52,7 +52,8 @@ export class BaseAction {
   public doesActionApply(vimState: VimState, keysPressed: string[]): boolean {
     if (
       this.mustBeFirstKey &&
-      vimState.recordedState.commandWithoutCountPrefix.length > keysPressed.length
+      (vimState.recordedState.commandWithoutCountPrefix.length > keysPressed.length ||
+        vimState.recordedState.operator)
```

Comment:
This is needed because the actions that must be first key can't come after an operator and since I'm resetting the commandList on `modeHandler` `runAction` function the `commandWithoutCountPrefix` might still have the same length so we need to check if there is no operator.